### ISSUE TITLE
Error when launching AntaresWebServer desktop

### ISSUE THREAD

**Issue by tbittar** [AntaresSimulatorTeam/AntaREST#1406]
## Description

When I launch AntaresWebServer desktop version, I get the following error :
```
plyer/platforms/linux/notification.py:65: UserWarning: The Python dbus package is not installed.
Try installing it with your distribution's package manager, it is usually called python-dbus or python3-dbus, but you might have to try dbus-python instead, e.g. when using pip.
libpng warning: iCCP: CRC error
Process Process-1:
Traceback (most recent call last):
  File "logging/config.py", line 563, in configure
  File "logging/config.py", line 744, in configure_handler
  File "logging/__init__.py", line 1147, in __init__
  File "logging/__init__.py", line 1176, in _open
FileNotFoundError: [Errno 2] Aucun fichier ou dossier de ce type: '/home/bittartho/logiciels/AntaresWeb-ubuntu-v2.13.0/AntaresWeb/tmp/antarest.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "multiprocessing/process.py", line 315, in _bootstrap
  File "multiprocessing/process.py", line 108, in run
  File "antarest/gui.py", line 21, in run_server
  File "antarest/main.py", line 121, in fastapi_app
  File "antarest/core/logging/utils.py", line 96, in configure_logger
  File "logging/config.py", line 808, in dictConfig
  File "logging/config.py", line 570, in configure
ValueError: Unable to configure handler 'default'
```

### Steps to reproduce the behavior:
1. Download AntaresWeb build (ubuntu version) at [https://github.com/AntaresSimulatorTeam/AntaREST/releases/download/v2.13.0/AntaresWeb-ubuntu-v2.13.0.zip](https://github.com/AntaresSimulatorTeam/AntaREST/releases/download/v2.13.0/AntaresWeb-ubuntu-v2.13.0.zip)
2. Unzip in a local directory on your laptop, let's call it `myAntaresWebDir`
3. Grant execution rights to AntaresWebServer, AntaresWebWorker, AntaresTool (I do not know which ones are really necessary): 
```bash
cd myAntaresWebDir
chmod +x AntaresWebServer
chmod +x AntaresWebWorker
cd AntaresWeb
chmod +x AntaresWebServer
chmod +x AntaresTool
```
4. Run AntaresWebServer (the exe that is located in the `AntaresWeb` subdirectory :
```bash
cd myAntaresWebDir/AntaresWeb
./AntaresWebServer
```
5. I get the aforementioned error.

## Expected Behavior

I expect AntaresWeb to launch normally.
## Screenshots

No relevant screenshots to add.
## Environment

- Personal laptop
- Ubuntu 20.04
- I tried running the command in a conda environment with all required packages from the error message :
```bash
conda list | grep dbus
dbus                      1.13.6               h5008d03_3    conda-forge
dbus-python               1.3.2           py310h253d92d_2    conda-forge
```

**Comment by makdeuneuv** [comment]
Here is the documentation to do the installation of the desktop version. You don't need to modify rights of the files.
[https://antares-web.readthedocs.io/en/updatedocs/#for-linux](https://antares-web.readthedocs.io/en/updatedocs/#for-linux 
)
Maybe if you try in this way it's would be work.

**Comment by MartinBelthle** [comment]
@tbittar this issue should be fixed with the new release 2.14.2. You can try with this new version and if it works close the issue :)

**Comment by tbittar** [comment]
Yes it works !
